pages/interview-process.md

```diff
@@ -3,62 +3,43 @@ permalink: /interview-process/
-	 - Knowledge of web framework(s) or other relevant frameworks
-	 - Demonstrated ability to write well-structured code
-	 - Understanding of web technology 
-	 - Understanding of modularity
+The interview process begins with a preliminary screening, a 30-minute phone or video call. Plan to talk about your skills and experience, what you're passionate about, and the work we do here.
 
--  Communication and collaboration skills
-
-	 - Ability to articulate technical concepts 
-	 - Ability to speak concisely and boldly
-	 - Understands working iteratively or as agile team
-	 - Teachability
-	 - Quick learning
-	 - Flexibility of thought
-
-- Leadership and entrepreneurship
-
-	 - Capable of evangelism
-	 - Can they teach us something we don’t know?
-	 - Will they “raise the bar” of our team?
+### In-person (or videochat) interviews
 
--  Creativity
+After your phone interview, the 18F member you spoke to will submit their notes to our Talent Team. If we'd like to chat with you more, we'll ask you to come in to one of our office locations or join us somewhere on the internet for a couple of longer interviews. Depending on what position you're applying for, the format will vary, but generally we group interviews with multiple candidates together into bi-weekly hiring sprints, every other Friday. 
 
-	 - Ability to translate creativity into practical solutions 
-	 - Change hearts and minds
+### Application qualification competencies
+During the qualification and evaluation process, we will assess you and your application against the following competencies:
+
+- **Analysis and creative problem solving:** Expertise determining user
+needs or identifying market opportunities and designing and
+deploying solutions.
+- **Technology acumen:** Knowledge of standard and emergent web
+technologies.
+- **Entrepreneurship and product management:** Experience delivering
+products or services from concept to market.
+- **Approach and methodology:** Expertise applying frameworks that
+emphasize delivery, iteration, quantitative and qualitative learnings,
+and responsiveness.
+- **Empathy and emotional intelligence:** Experience in client-facing
+roles, navigating bureaucracy, business development, or delivering
+services with a strong user-facing or consumer-oriented element.
+- **Tools and execution:** Experience in design, development and
+deployment of solutions leveraging current and emergent
+technologies and best practices.
 
 ### Interviewing for a specific team? 
-
 Check out more interviewing information for each of our teams. 
 
 <ul>
-	{% for interview in site.interviews %}
-		<li><a href="{{site.baseurl}}{{ interview.url }}">{{ interview.title }}</a></li>
-	{% endfor %}
+{% for interview in site.interviews %}
+<li><a href="{{site.baseurl}}{{ interview.url }}">{{ interview.title }}</a></li>
+{% endfor %}
 </ul>
-
-Don't see the team you're interviewing with? Design, content, and operations information will be added soon. 	
```

Comment:
@rooneywp Reducing user anxiety FTW. :) Would you prefer a more general phrasing?